.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: go
+sudo: false
+go:
+    - 1.7
+install:
+    - cd cluster-state-service; make get-deps; cd ../
+    - cd daemon-scheduler; make get-deps; cd ../
+script:
+    - cd cluster-state-service; make; make release; cd ../
+    - cd daemon-scheduler; make; make release; cd ../
```

Comment:
Same as above. Can you change `release` to `unit-tests`? If `make` already covers that, you wouldn't need the second `make` anyway. Also, why is `cd ../` required here?